tfac/dataHelpers.py

```diff
@@ -21,44 +21,16 @@ def importLINCSprotein():
 
     return pd.concat([dataA, dataB, dataC])
 
-
-def ohsu_data(export=False):
-    """ Import OHSU data for PARAFAC2"""
-    RNAseq = pd.read_csv(join(path_here, "tfac/data/ohsu/MDD_RNAseq_Level4.csv"), delimiter=",", index_col=0)
-
-    row_avg = RNAseq.mean(axis=1)
-    for indx in RNAseq.index:
-        if row_avg[indx] <= 0.07:
-            RNAseq.drop(indx, inplace=True)
-
-    # column names
-    cols = RNAseq.columns
-    if export:
-        RNAseq = RNAseq.apply(scale, axis=1, result_type='expand')
-        RNAseq.columns = cols
-        RNAseq.to_csv(join(path_here, "tfac/data/ohsu/RNAseq.txt"), sep='\t')
-
-    return RNAseq
-
-
 def proteinNames():
     """Return protein names (data columns)"""
     data = importLINCSprotein()
     data = data.drop(columns=["Treatment", "Sample description", "File", "Time"], axis=1)
     return data.columns.values.tolist()
 
+def process_proteins():
+    """ import, group, and scale proteins into a dataframe, ready to be merge with RNAseq into a tensor. """
 
-def reorder_table(df):
```

Comment:
Why did this function move?

Reply:
no important reason -- I wanted the function that were working on proteins to be placed together, and those working on genes also together.. this one was a bit out of order so I placed it at the end.